packages/features/form-builder/Components.tsx

```diff
@@ -310,7 +310,7 @@ export const Components: Record<BookingFieldType, Component> = {
                         type="radio"
                         disabled={readOnly}
                         name={name}
-                        className="dark:bg-darkgray-300 border-subtle text-emphasis h-4 w-4 focus:ring-black ltr:mr-2 rtl:ml-2"
+                        className="dark:checked:bg-brand-default dark:bg-darkgray-100 dark:hover:bg-subtle dark:checked:hover:bg-brand-default focus:border-brand-default focus:ring-brand-default border-emphasis h-4 w-4 cursor-pointer text-[--cal-brand] focus:ring-2 ltr:mr-2 rtl:ml-2"
```

Comment:
```suggestion
                        className="dark:checked:bg-brand-default dark:bg-darkgray-100 dark:hover:bg-subtle dark:checked:hover:bg-brand-default focus:border-brand-default focus:ring-brand-default border-emphasis h-4 w-4 cursor-pointer text-brand focus:ring-2 ltr:mr-2 rtl:ml-2"
```

NIT I think we need to double check the classes here - Not using tokens in a few places

Reply:
When using `text-brand` the checkbox turns white. Please check this out
https://www.loom.com/share/2f0bdc127bdb4221b96660f459d6d46c

Reply:
Interesting okay - lets just leave it as it is for now and stick with the css variable. 

I will look into this in another PR